test/Engine/ProtoTestFx/WatchTestFx.cs

```diff
@@ -140,7 +140,7 @@ protected override void POP_Handler(Instruction instruction)
             {
                 int lineNo = instruction.debug.Location.StartInclusive.LineNo;
 
-                if (Core.Options.IDEDebugMode && Core.ExecMode != ProtoCore.DSASM.InterpreterMode.kExpressionInterpreter)
```

Comment:
Yea it should have been
I will move this in http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6193
This is part of this sprint